### DISCUSSION TITLE
There is a new release v3.27.0 (v3.27.0) created 24 days ago

### DISCUSSION THREAD

**Discussion by Andystew94** [qdm12/gluetun#852] [Q&A] [answered]
Hi mate, I just downloaded the "latest" version of gluetun and deployed it. However, it has reported that it is not the latest?

As per the logs:
2022/02/17 21:18:31 INFO vpn: There is a new release v3.27.0 (v3.27.0) created 24 days ago

![image](https://user-images.githubusercontent.com/25321854/154471399-5d5188f6-4aa3-486c-898b-e6829121a846.png)

![image](https://user-images.githubusercontent.com/25321854/154471523-1c713bb8-0411-4ef0-ade1-7cc9c87532f6.png)

**Comment by qdm12** [comment, marked as answer]
Hi there!

I do releases more or less regularly when gluetun has been looking stable for some time. So the latest release v3.27.0 is the last stable with image `qmcgaw/gluetun:v3.27.0`. Now the one you're running 'latest' is the image corresponding to the latest development and can be sometimes unstable, but you have the latest features/fixes as well, and you provide critical feedback for me as well if something goes wrong for the next release 😉

EDIT: on the other hand, I'll add a log message saying you're running N commits after the latest release to make it clear you're not lagging behind.

**Reply by qdm12** [reply]
Actually @Andystew94 this is already done, if you are running the latest qmcgaw/gluetun image it should say you're running version latest and how many commits you are behind the last commit.

What did you get logged regarding your version in the logs (`You are running version xxx ...` line)?

**Reply by Andystew94** [reply]
Apologies mate. I just updated the container to your "latest" and this issue has resolved itself.

**Reply by Andystew94** [reply]
The version which produced this irregularity was the previous commit. 
I make a habit of updating this container each time duin alerts me to a newer version.

**Reply by qdm12** [reply]
Yes but that's not the expected behavior, even if you run an older latest, it should show you are running latest and that you are lagging N commits behind the last commit. It should not log things about the last release. The last release log should only be shown if you run a version different than `latest`. So it might be a bug in my build pipeline :smile: Anyway if you see it again, feel free to report it! Thanks!
